lookout/style/format/features.py

```diff
@@ -401,10 +401,13 @@ def _get_left_sibling_features(self, left_sibling_vnode: VirtualNode, vnode: Vir
                 abs(left_sibling_vnode.end.line - vnode.end.line),
                 left_sibling_vnode.start.col - vnode.start.col,
                 left_sibling_vnode.end.col - vnode.end.col,
+                left_sibling_vnode.end.offset - left_sibling_vnode.start.offset,
```

Comment:
isn't it a duplication of ```left_sibling_vnode.start.col - vnode.start.col, left_sibling_vnode.end.col - vnode.end.col```?

Reply:
No, this is the relative position of the neighbor compare to the node while my addition is just the length of the neighbor.